File: CRM/src/app/shared/eHospital_DongNai_A_Config/Sys_AppPrivateSettingItems.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_AppPrivateSettingItems } from './Sys_AppPrivateSettingItems.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_AppPrivateSettingItemsService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Setting_Id', 'SettingCode', 'SettingName', 'ControlName'];

    List: Sys_AppPrivateSettingItems[] | undefined;
    ListFilter: Sys_AppPrivateSettingItems[] | undefined;
    FormData!: Sys_AppPrivateSettingItems;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_AppPrivateSettingItems";
    }
}

File: Service_eHospital_DongNai_A_Config/Implement/Sys_AppPrivateSettingItemsService.cs
namespace Service_eHospital_DongNai_A_Config.Implement
{
    public class Sys_AppPrivateSettingItemsService : BaseService<Sys_AppPrivateSettingItems, ISys_AppPrivateSettingItemsRepository>
    , ISys_AppPrivateSettingItemsService
    {
    private readonly ISys_AppPrivateSettingItemsRepository _Sys_AppPrivateSettingItemsRepository;
    public Sys_AppPrivateSettingItemsService(ISys_AppPrivateSettingItemsRepository Sys_AppPrivateSettingItemsRepository) : base(Sys_AppPrivateSettingItemsRepository)
    {
    _Sys_AppPrivateSettingItemsRepository = Sys_AppPrivateSettingItemsRepository;
    }
    }
    }

